src/routes/Routes.tsx

```diff
@@ -173,9 +173,4 @@ const Routes = () => {
   );
 };
 
-// We need `withRouter` here because (surprisingly) react-router-dom is not
-// using the router context property (available to e.g. each `Switch`):
-// https://github.com/ReactTraining/react-router/issues/4671
-// We are using `withRouter` to get around `connect()`'s `shouldComponentUpdate`
-// function blocking updates when the route location changes.
-export default withRouter(Routes);
+export default Routes;
```

Comment:
I think this was needed because we used `react-router` directly sometimes? Or the connected redux-router? I don't know, it does work now.